modules/multicast/player.c

```diff
@@ -560,3 +560,22 @@ void mcplayer_stop(void)
 {
 	player = mem_deref(player);
 }
+
+
+/**
+ * Initialize everything needed for the player beforhand
+ *
+ */
+void mcplayer_init(void)
```

Comment:
in general an init function should return `int` (errno) because it can potentially fail.